### DISCUSSION TITLE
Philips Hue Gradient Lightstrip 55 - not recognised

### DISCUSSION THREAD

**Discussion by DonRonito** [Koenkk/zigbee2mqtt#11495] [Devices]
Hi all,

Adding a Gradient Lightstrip 55, it's not recognised. Seeing that the 65 is available (https://www.zigbee2mqtt.io/devices/929002422801.html), it there a way for manually selecting that config for the 55 version?

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+

**Reply by DonRonito** [reply]
Thanks. Will add this tonight.
